### ISSUE TITLE
Typescript error on managing customer groups on Medusa Admin

### ISSUE THREAD

**Issue by brendansimcox** [medusajs/medusa#5228]
### Describe the bug

Medusa Admin is throwing a typescript error t is undefined on editing a customer group in the latest medusa admin panel

### System information

Medusa version (including plugins): 1.16.1
Node.js version: 16
Database: Postgres
Operating system: MacOS
Browser (if relevant): Chrome

### Steps to reproduce the behavior

1. Log in to admin panel
2. Go to customers, click groups
3. Select an existing, or create a new customer group then select that one

### Expected behavior

Edit screen should render without application error

### Screenshots

![image](https://github.com/medusajs/medusa/assets/90048338/a7de6d4d-4719-42a0-a1fb-85992c5758f1)

### Code snippets

```
ReferenceError: t is not defined
    at iw (https://hiddenurl.co.uk/app/276.1e191f19.chunk.js:1:206954)
    at oO (https://hiddenurl.co.uk/app/main.39157b15.js:18:60913)
    at ah (https://hiddenurl.co.uk/app/main.39157b15.js:18:72795)
    at u (https://hiddenurl.co.uk/app/main.39157b15.js:18:120331)
    at sI (https://hiddenurl.co.uk/app/main.39157b15.js:18:99087)
    at https://hiddenurl.co.uk/app/main.39157b15.js:18:98954
    at sN (https://hiddenurl.co.uk/app/main.39157b15.js:18:98961)
    at sP (https://hiddenurl.co.uk/app/main.39157b15.js:18:95713)
    at s_ (https://hiddenurl.co.uk/app/main.39157b15.js:18:96102)
    at n3 (https://hiddenurl.co.uk/app/main.39157b15.js:18:44823)

```

**Comment by olivermrbl** [comment]
Solved in #5162

**Comment by olivermrbl** [comment]
Will be part of next releasing going out this Friday :)
